### ISSUE TITLE
could not get qdiscs: unsupported kind: fq

### ISSUE THREAD

**Issue by itzloop** [livekit/livekit#1417]
Hello,

**Describe the bug**
I tried to deploy livekit-server on Kubernetes and got this error:
```
ERROR   livekit routing/redisrouter.go:446      error processing RTC message    {"error": "could not get qdiscs: unmarshalFq()\t13\n\t[16 39 0 0]\nextractXStats(): unsupported kind: fq"}
```
I don't think this issue is a bug of this project since everything was fine with this setup in a VM so it has to be the environment. However i am clueless and don't know where to look so any help will be appreciated. 

**Server**
- Version: v1.3.3 and also v1.3.2
- Environment: Bare  metal Kubernetes cluster
- any other information about your deployment setup
Used the helm chart that was provider. Modified the ingress a little bit.

Thank you for this awesome project. :v:

**Comment by bekriebel** [comment]
I had this issue as well. I'm not sure what the root cause is; I was running on a VPS and not k8s directly. A change was put in recently that doesn't fix the stats, but does make it so the server doesn't crash when it fails to pull the stats. Unfortunately, it was merged after the latest release: https://github.com/livekit/livekit/pull/1386.

If you'd like to use it, I built a docker image that has this change in place. https://github.com/bekriebel/livekit/pkgs/container/livekit-server/69902171?tag=main-481b4ab
